packages/components/pie-notification/src/defs.ts

```diff
@@ -1,3 +1,34 @@
-// TODO - please remove the eslint disable comment below when you add props to this interface
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface NotificationProps {}
+export const variants = ['neutral', 'neutral-alternative', 'info', 'success', 'warning', 'error'] as const;
+export const headingLevels = ['h2', 'h3', 'h4', 'h5', 'h6'] as const;
+
+export interface NotificationProps {
+  /**
+   * Set the variant of the notification.
+   */
+  variant: typeof variants[number];
```

Comment:
This property should probably be optional too given that it has a default value. Otherwise I believe react TS consumers would still need to provide it